src/manage_nsfs/health.js

```diff
@@ -450,15 +450,17 @@ class NSFSHealth {
      */
     _get_config_dir_status(system_data) {
         if (!system_data) return { error: 'system data is missing' };
+        const blocked_hosts = this._get_blocked_hosts_status(this.config_fs, system_data);
         const config_dir_data = system_data.config_directory;
-        if (!config_dir_data) return { error: 'config directory data is missing, must upgrade config directory' };
+        if (!config_dir_data) return { error: 'config directory data is missing, must upgrade config directory', blocked_hosts };
```

Comment:
I don't see the `blocked_hosts` in the PR description:

> 5. expect to see the following blocked hosts in the health response -
> "config_directory_status": {
>           "error": "config directory data is missing, must upgrade config directory",
>           "blocked_hosts": {
>             "hostname1": {
>               "host_version": "5.18.0",
>               "host_config_dir_version": "1.0.0",
>               "error": "host's config_dir_version is 1.0.0, system's config_dir_version is undefined"
>             }
>           }
>         }

Reply:
not sure I understand, I see it in the reference you added

Reply:
I don't remember what I meant here, you're right I can see the `blocked_hosts` as well.